src/metabase/lib/util.cljc

```diff
@@ -330,46 +330,6 @@
            conjunction
            (last coll)))))))
 
-(defn update-stages-ignore-joins
```

Comment:
Unrelated but these were unused and didn't actually work right, I thought I removed them in a different PR but I guess I didn't, so I'm removing them here. We can reimplement them in the future with tests if we actually need them for something.